fuzz/fuzz_targets/read_region.rs

```diff
@@ -0,0 +1,14 @@
+#![no_main]
+use libfuzzer_sys::fuzz_target;
+
+use fastanvil::RegionBuffer;
+use std::io::Cursor;
+
+fuzz_target!(|data: &[u8]| {
+    // fuzzed code goes here
+    let reader = Cursor::new(data);
+    // TODO: this function expects the input to be compressed, find a way to avoid that
+    let r = RegionBuffer::new(reader);
+    let _location = r.chunk_location(0, 0);
+    let _chunk = r.load_chunk(0, 0);
+});
```

Comment:
In this target we are basically fuzzing the implementation of zlib, which is not ideal.